### ISSUE TITLE
[BUG] Issue to make prediction on CPU using a GPU-trained model, after doing the correct loading, using map_location + .to_cpu())

### ISSUE THREAD

**Issue by simonweppe** [unit8co/darts#1694]
Hi there,
With the infos provided here: https://github.com/unit8co/darts/issues/1624 I was able to properly load a model trained on GPU to be used on CPU.

However, when I try to predict Im getting using that loaded model I get :

```
model_TCN = TCNModel.load(mod_name, map_location = 'cpu') 
model_TCN.to_cpu()
pred = model_TCN.predict(30, series=train_trf, past_covariates=None)


*** ValueError: 
 you tried to log -1 which is currently not supported. Try a dict or a scalar/tensor.
```

Any suggestions on what I might be ?
Could be related to this: https://github.com/Lightning-AI/lightning/issues/4070 ..but not too sure. 
Thanks

**Comment by madtoinou** [comment]
The error indeed correspond to what is described in the other issue. By any chance, did you update your environment recently (which version of PytorchLightning are you using?).

They suggest to install tensorboard and delete the log folder ("darts_logs in this contest), did you try these two solutions?

**Comment by simonweppe** [comment]
Thanks @madtoinou.
No recent update,

```
>>> darts.__version__
'0.23.1'

>>> torch.__version__
'1.13.1+cu117'
```

Yep I tried to delete darts_logs, and I do have tensorboard. Played a bit with all permissions as well but no luck so far.

I've just found a workaround doing a quick dummy training on the GPU machine.
If I set `log_tensorboard = False` when initializing the model, then train it like that on the GPU, and save, then I can correctly use it on the CPU machine without that error. So definitely something around tensorboard/logs ...

Not ideal, as it's nice to have the logs obviously, but that can get me through for now.
Thanks for the help
